froide_payment/provider/stripe.py

```diff
@@ -750,6 +760,29 @@ def confirm_single_payment(self, payment):
         payment.change_status(PaymentStatus.PENDING)
         payment.save()
 
+    def customer_subscription_created(self, request, subscription):
+        customer = Customer.objects.get(remote_reference=subscription.customer)
+        plan = Plan.objects.get(remote_reference=subscription.plan.id)
```

Comment:
`subscription` ist hier ein Stripe Objekt – und kein Django model. Es würde hier helfen die Variable in `stripe_subscription` umzubenennen und ggf. type zu annotieren.